prac2-pages/anshumali_suri/lib/main.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+
+      home: Scaffold(
+
+        body: const MyStatefulWidget(),
+
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  TextEditingController nameController = TextEditingController();
```

Comment:
A better way to initialize the TextEditingControllers is through overriding the initState() method, and then you must free the resources through overriding the dispose() method. 

https://api.flutter.dev/flutter/widgets/TextEditingController-class.html
**Please refer to the above documentation for dispose() method**

Reply:
Resolved in commit [Update main.dart](https://github.com/RitwikSingh28/GitHubPrac/pull/15/commits/7d63d65c799f15f3d81c7dde0ead33dff3ae0ba8)